packages/api/prisma/schema.prisma

```diff
@@ -10,62 +10,102 @@ generator client {
 datasource db {
   provider   = "postgresql"
   url        = env("DATABASE_URL")
+  // pgcrypto is necessary for ULIDs which are better than random UUIDs.
   extensions = [pgcrypto]
 }
 
+/// A book of the Bible.
 model Book {
+  /// The ID of the book. Corresponds to its order in the Bible.
   id     Int     @id
+  /// The three letter code used to for look ups from references.
   name   String  @unique
+  /// The list of verses in the book.
   verses Verse[]
```

Comment:
I added comments to describe each table and field. Comments with three slashes can later be used by a prisma generator to generate docs for our schema. For now I think linking to the schema file in our markdown docs should be sufficient.